test/api-url.test.js

```diff
@@ -0,0 +1,116 @@
+import { fixture, assert, nextFrame, html } from '@open-wc/testing';
```

Comment:
Please, create `d.ts` file for this one and update `d.ts` file for the method documentation. With compatible IDE with TS support you get a lot when developing with types definitions. Add `@attribute` annotation to scalar properties and `@fires` attribute to the class is an event is dispatched. It helps with linting whether you are using the right attributes when consuming the component. See more here: https://github.com/runem/lit-analyzer/tree/master/packages/lit-analyzer